### DISCUSSION TITLE
Changes are not visible in frontend code

### DISCUSSION THREAD

**Discussion by mayank-genix** [Shuffle/Shuffle#1490] [Q&A]
I have recently started working on shuffle code. I am using docker compose , website is working fine but if i am trying to make changes in the frontend code it is not reflecting .

**Comment by frikky** [comment]
> I have recently started working on shuffle code. I am using docker compose , website is working fine but if i am trying to make changes in the frontend code it is not reflecting .

Hola! Are you running the frontend itself in docker? If so, that won't work. You have to run the frontend server as it's built in react :)

**Comment by Rezanmonis** [comment]
Hello

We are looking for frontend and backend developers as our teammates for our edtech project rezanmonis@gmail.com
